core/src/calls.rs

```diff
@@ -1,22 +1,18 @@
 use std::sync::Arc;
 use types::{Params, Value, Error};
-use futures::Future;
+use futures::{Future, IntoFuture};
 use BoxFuture;
 
 /// Metadata trait
 pub trait Metadata: Default + Clone + Send + 'static {}
 impl Metadata for () {}
 
-/// Synchronous Method
-pub trait RpcMethodSync: Send + Sync + 'static {
-	/// Call method
-	fn call(&self, params: Params) -> Result<Value, Error>;
-}
-
 /// Asynchronous Method
 pub trait RpcMethodSimple: Send + Sync + 'static {
+	/// Output future
+	type Out: Future<Item = Value, Error = Error> + Send;
```

Comment:
why not bound this by `IntoFuture` also?

Reply:
1. Consistency: `RpcMethod:call` returns `Future` as well
2. `Send` bound